src/operator/nn/concat.cc

```diff
@@ -166,18 +168,17 @@ bool ConcatType(const nnvm::NodeAttrs& attrs,
     for (size_t i = 0; i < nin; ++i) {
       in_type->push_back(dtype);
     }
-  // if out types are known in types are unknown
+    // if out types are known in types are unknown
```

Comment:
I like this suggestion:
https://stackoverflow.com/a/64052524/5008804
but on the other hand it's troublesome to change them. maybe we should not be so strict about it.

Reply:
Sounds good! I'm trying to check what could have been done to prevent this situation. [Workaround] First of all, we can try to add additional space between `if {} else {}` scope. However, It forces each of us to care about that ~ what sounds not bad! Secondly, a comment might be placed directly in a line with a code.